client/src/services/fileOperationManager.ts

```diff
@@ -687,6 +732,64 @@ class FileOperationManager {
     await this.sendState(FileOperationState.EntryRestored, data);
   }
 
+  private async doDownload(data: DownloadData): Promise<void> {
```

Comment:
Is there a reason not to consider `MIN_OPERATION_TIME_MS` ?

Reply:
Forgot about it, it's fixed.